books/free-programming-books-it.md

```diff
@@ -72,6 +73,14 @@
 * [Storia dell'informatica](http://www.dsi.unive.it/~pelillo/Didattica/Storia%20dell'informatica/) - M. Pelillo (PDF)
 
 
+#### AngularJS
+
+> :information_source: Vedi anche &#8230; [Angular](#angular)
+> 
+* [AngularJS, il supereroe dei framework JavaScript ...di Google](https://www.html.it/articoli/angularjs-il-supereroe-dei-framework-javascript-di-google/) - Andrea Chiarelli (HTML)
+* [Guida AngularJS](https://www.html.it/guide/guida-angularjs/) - Andrea Chiarelli (HTML)
+
+
```

Comment:
Hey!! Why transfer this from lines 184-191 to here? Please, revert

Reply:
Sorry, I have take this part from an old fork. I will correct it now